modules/salt/pages/large-scale.adoc

```diff
@@ -174,3 +174,15 @@ If significant non-transient swap usage is detected, you can increase the availa
 
 You can also consider tuning the Server to consume less memory.
 For more information on tuning, see xref:salt:large-scale-tuning.adoc[].
+
+=== AES Key rotation
+
+Communication from the Salt Master to all Salt minions is encrypted with a single AES key. This key is rotated whenever:
```

Comment:
```suggestion
Communications between the Salt Master and clients is encrypted with a single AES key. 
The key is rotated when:
```

Reply:
Hmm, not really, in the sense communication is not bidirectionally encrypted in the same way. Here I am really referring to messages from master to minions only.

Reply:
Ah, my bad. What about this:

```suggestion
Communications from the Salt Master to clients is encrypted with a single AES key.
The key is rotated when:
```